packages/notifier/src/asyncIterableAdaptor.js

```diff
@@ -34,59 +36,7 @@ import './types-ambient.js';
  * @param {ERef<BaseNotifier<T>>} notifierP
  * @returns {ConsistentAsyncIterable<T>}
  */
-export const makeAsyncIterableFromNotifier = notifierP => {
-  return Far('asyncIterableFromNotifier', {
-    [Symbol.asyncIterator]: () => {
-      /** @type {UpdateCount} */
-      let localUpdateCount;
-      /** @type {Promise<{value: T, done: boolean}> | undefined} */
-      let myIterationResultP;
-      return Far('asyncIteratorFromNotifier', {
-        next: () => {
-          if (!myIterationResultP) {
-            // In this adaptor, once `next()` is called and returns an
-            // unresolved promise, `myIterationResultP`, and until
-            // `myIterationResultP` is fulfilled with an
-            // iteration result, further `next()` calls will return the same
-            // `myIterationResultP` promise again without asking the notifier
-            // for more updates. If there's already an unanswered ask in the
-            // air, all further asks should just reuse the result of that one.
-            //
-            // This reuse behavior is only needed for code that uses the async
-            // iterator protocol explicitly. When this async iterator is
-            // consumed by a for/await/of loop, `next()` will only be called
-            // after the promise for the previous iteration result has
-            // fulfilled. If it fulfills with `done: true`, the for/await/of
-            // loop will never call `next()` again.
-            //
-            // See
-            // https://2ality.com/2016/10/asynchronous-iteration.html#queuing-next()-invocations
-            // for an explicit use that sends `next()` without waiting.
-            myIterationResultP = E(notifierP)
-              .getUpdateSince(localUpdateCount)
-              .then(({ value, updateCount }) => {
-                localUpdateCount = updateCount;
-                const done = localUpdateCount === undefined;
-                if (!done) {
-                  // Once the outstanding question has been answered, stop
-                  // using that answer, so any further `next()` questions
-                  // cause a new `getUpdateSince` request.
-                  //
-                  // But only if more answers are expected. Once the notifier
-                  // is `done`, that was the last answer so reuse it forever.
-                  myIterationResultP = undefined;
-                }
-                return harden({ value, done });
-              });
-          }
-          // xxx hint for type checker
-          assert(myIterationResultP);
-          return myIterationResultP;
-        },
-      });
-    },
-  });
-};
+export const makeAsyncIterableFromNotifier = subscribeLatest;
```

Comment:
If these are synonyms, can we deprecate and/or remove one? Preferably remove `makeAsyncIterableFromNotifier`?

Reply:
Line 9 marks the deprecation.